__tests__/app.test.js

```diff
@@ -170,4 +170,72 @@ describe("/api/articles/:article_id/comments", () => {
         });
     });
   });
-});
+  describe("POST", () => {
+    test("status: 201 inserts a new comment to an article with correct properties", () => {
+      const newComment = {
+        username: "butter_bridge",
+        body: "test comment 1",
+      };
+      return request(app)
+        .post("/api/articles/3/comments")
+        .send(newComment)
+        .expect(201)
+        .then((response) => {
+          expect(typeof response.body).toBe("object");
+          expect(response.body.comment.body).toBe("test comment 1");
+          expect(response.body.comment.author).toBe("butter_bridge")
+          expect(response.body.comment.article_id).toBe(3)
+          expect(typeof response.body.comment.comment_id).toBe("number")
+          expect(typeof response.body.comment.votes).toBe("number")
+          expect(typeof response.body.comment.created_at).toBe("string")
```

Comment:
Using objectContaining matcher would be better when testing objects.

Jest documents have lots of matchers that would be worth looking into